scss/_card.scss

```diff
@@ -4,6 +4,7 @@
 
 .card {
   // scss-docs-start card-css-vars
+  --#{$prefix}card-spacer-top: #{$card-spacer-top}; // Boosted mod
```

Comment:
IMO it's disturbing to have a `--bs-card-spacer-top` that acts on the top padding and then something named `--bs-card-spacer-y` only used for the bottom padding.
As a user, I don't know how to use those variables; will `--bs-card-spacer-top` override the value of the `--bs-card-spacer-y`, which one has the priority, etc.

Reply:
Another solution could be to set the padding top as a ratio of the `card-spacer-y`